src/organizercore.cpp

```diff
@@ -469,12 +469,14 @@ void OrganizerCore::updateVFSParams(log::Levels logLevel,
                                     env::CoreDumpTypes coreDumpType,
                                     const QString& crashDumpsPath,
                                     std::chrono::seconds spawnDelay,
-                                    QString executableBlacklist)
+                                    QString executableBlacklist,
+                                    const QString& skipFileSuffixes,
```

Comment:
I will do a more thorough review later on but globally, I think you should change to `QStringList` instead of `QString` in most places to handle the list of file suffixes / directory names to skip. I guess you did what was done for the executable black list (I'm not familiar with the implementation so I do not know how it's handled), but I do not see any reason not to use `QStringList` here (`QSettings` can perfectly handle `QStringList`).

Reply:
Made the swap to `QStringList`, ready for review again